swap/swap.go

```diff
@@ -252,10 +272,24 @@ func (swap *Swap) GetMetricTypes(cfg plugin.ConfigType) ([]plugin.MetricType, er
 		metricType := plugin.MetricType{Namespace_: core.NewNamespace(vendorPrefix, srcPrefix, typePrefix, ioPrefix, metric)}
 		metricTypes = append(metricTypes, metricType)
 	}
+	// List of terminal metric names
+	mList := make(map[string]bool)
 	for _, device := range devices {
```

Comment:
That makes sense. I think it is fine to forego it for now because you'll be changing it in a future PR